README.md

```diff
@@ -4,4 +4,4 @@ This is a small Ember application that allows both researchers to preview an exp
 participate in an experiment. This is meant to be used in conjunction with the [Lookit API Django project](https://github.com/lookit/lookit-api), which contains the Experimenter and Lookit applications.
 The Django applications will proxy to these Ember routes for previewing/participating in an experiment.
 
-For installation instructions, please see the [Lookit documentation](https://lookit.readthedocs.io/en/develop/ember-app-installation.html). 
+For installation instructions, please see the [Lookit documentation](https://lookit.readthedocs.io/projects/frameplayer/en/latest/). 
```

Comment:
This is the working link to the EFP docs 👍 
But I wonder if the intent in this readme is to link to the EFP _installation_ docs? If so, we could use this link instead:
https://lookit.readthedocs.io/en/develop/install-ember-app.html
(Note the change is just in the last part of the link: `ember-app-installation` -> `install-ember-app`)